src/common.js

```diff
@@ -246,12 +246,6 @@ function parseOptions(opt, map, errorOptions) {
 }
 exports.parseOptions = parseOptions;
 
-function globOptions() {
```

Comment:
I think this was reverted when it should not have been. I think what happened is that you probably have an out-of-date copy of the `master` branch. Can you please sync this against the main repo's master branch? I think you can do this with something like:

```
git checkout master
git remote add upstream https://github.com/shelljs/shelljs.git
git fetch upstream
git rebase upstream/master
git checkout switch-fast-glob
git rebase master
git push origin switch-fast-glob --force-with-lease
```